test/validation/transcriptions/invalid/unescaped-heading.txt

```diff
@@ -0,0 +1,11 @@
+*Image Transcription:*
+
+---
+
+Blah blah
+
+#I hate bugs
```

Comment:
Can you add a space between the `#` and the `I hate bugs` here? Otherwise it will also return the unescaped heading formatting issue soon

Reply:
Wait I'm dumb, I thought I was on a different PR